scripts/minishift-setup.sh

```diff
@@ -0,0 +1,66 @@
+
+#!/bin/bash
+#
+#
+
+set -euo pipefail
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+cd ..
+
+# Lets try to detect the local machine ip..
+LOCAL_IP="$(nslookup $(hostname) | tail -n +4 | grep Address | cut -d " " -f 2)"
+LOCAL_IP="${1:-$LOCAL_IP}"
+
+sed "s/192.168.64.2/$(minishift ip)/" src/config.json.minishift > src/config.json
+oc replace --force -f - <<EOF
+{
+  "kind": "List",
+  "apiVersion": "v1",
+  "items": [
+    {
+      "apiVersion": "v1",
+      "kind": "Service",
+      "metadata": {
+        "labels": {
+          "app": "syndesis",
+          "component": "syndesis-ui"
+        },
+        "name": "syndesis-ui"
+      },
+      "spec": {
+        "ports": [
+          {
+            "port": 80,
+            "protocol": "TCP",
+            "targetPort": 4200,
+            "name": "http"
+          }
+        ],
+        "selector": {}
+      }
+    },
+    {
+      "apiVersion": "v1",
+      "kind": "Endpoints",
+      "metadata": {
+        "name": "syndesis-ui"
+      },
+      "subsets": [
+        {
+          "addresses": [
+            {
+              "ip": "${LOCAL_IP}"
```

Comment:
I love this idea! Use an external service! Nice thinking :bat:man

Reply:
And can't be the same behaviour achieved by `NodePort` [service](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport)?

Reply:
That might also work and get rid of the need to specify the IP address at all - genius!

Reply:
Ah... node would be the minishift VM, not the local dev machine.